plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestCheckpointEntryIterator.java

```diff
@@ -57,6 +66,7 @@
 {
     private static final String TEST_CHECKPOINT = "databricks/person/_delta_log/00000000000000000010.checkpoint.parquet";
 
+    private final TypeManager typeManager = TESTING_TYPE_MANAGER;
```

Comment:
You can just use TESTING_TYPE_MANAGER directly, without the extra variable definition here